<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
<head>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Unit Test Results: All Errors</title>
<link rel="stylesheet" type="text/css" title="Style" href="stylesheet.css">
</head>
<body onload="open('allclasses-frame.html','classListFrame')">
<h1>Unit Test Results.</h1>
<table width="100%">
<tr>
<td align="left"></td><td align="right">Designed for use with <a href="http://www.junit.org/">JUnit</a> and <a href="http://ant.apache.org/">Ant</a>.</td>
</tr>
</table>
<hr size="1">
<h2>All Errors</h2>
<table class="details" border="0" cellpadding="5" cellspacing="2" width="95%">
<tr valign="top">
<th>Class</th><th>Name</th><th>Status</th><th width="80%">Type</th><th nowrap>Time(s)</th>
</tr>
<tr valign="top" class="Error">
<td><a href="ptolemy/actor/lib/jjs/modules/mqtt/test/junit/60_JUnitTclTest.html">JUnitTclTest</a></td><td><a name="[0] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/mqtt/test/auto/MQTTPubSub.xml (RunModel)"></a><a href="ptolemy/actor/lib/jjs/modules/mqtt/test/junit/60_JUnitTclTest.html#[0] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/mqtt/test/auto/MQTTPubSub.xml (RunModel)">[0] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/mqtt/test/auto/MQTTPubSub.xml (RunModel)</a></td><td>Error</td><td>N/A<br>
<br>
<code>java.lang.reflect.InvocationTargetException<br>	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)<br>	at ptolemy.util.test.junit.AutoTests.RunModel(AutoTests.java:193)<br>	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)<br>Caused by: ptolemy.kernel.util.IllegalActionException: Exceptions occurred during wrapup:<br>ptolemy.kernel.util.IllegalActionException: The fire() method of this actor was never called. Usually, this is an error indicating that starvation is occurring.<br>  in .MQTTPubSub.Test<br>
<br>  in .MQTTPubSub.DE Director<br>Because:<br>The fire() method of this actor was never called. Usually, this is an error indicating that starvation is occurring.<br>  in .MQTTPubSub.Test<br>	at ptolemy.actor.Director.wrapup(Director.java:1736)<br>	at ptolemy.domains.de.kernel.DEDirector.wrapup(DEDirector.java:1527)<br>	at ptolemy.actor.CompositeActor.wrapup(CompositeActor.java:2755)<br>	at ptolemy.actor.Manager.wrapup(Manager.java:1512)<br>	at ptolemy.actor.Manager.execute(Manager.java:407)<br>	at ptolemy.actor.Manager.run(Manager.java:1263)<br>	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)<br>Caused by: ptolemy.kernel.util.IllegalActionException: The fire() method of this actor was never called. Usually, this is an error indicating that starvation is occurring.<br>  in .MQTTPubSub.Test<br>	at ptolemy.actor.lib.NonStrictTest.wrapup(NonStrictTest.java:394)<br>	at ptolemy.actor.Director.wrapup(Director.java:1722)<br>
</code></td><td>42.783</td>
</tr>
<tr valign="top" class="Error">
<td><a href="ptolemy/actor/lib/jjs/modules/serial/test/junit/64_JUnitTclTest.html">JUnitTclTest</a></td><td><a name="[4] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackShort.xml (RunModel)"></a><a href="ptolemy/actor/lib/jjs/modules/serial/test/junit/64_JUnitTclTest.html#[4] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackShort.xml (RunModel)">[4] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackShort.xml (RunModel)</a></td><td>Error</td><td>N/A<br>
<br>
<code>java.lang.reflect.InvocationTargetException<br>	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)<br>	at ptolemy.util.test.junit.AutoTests.RunModel(AutoTests.java:193)<br>	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)<br>Caused by: ptolemy.kernel.util.IllegalActionException: Failure executing the react function: Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1719)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1764) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1731 at column number 24<br>  in .SerialLoopbackShort.TrainableTest<br>Because:<br>Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1719)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1764) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1731 at column number 24<br>	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2306)<br>	at ptolemy.actor.lib.jjs.JavaScript.fire(JavaScript.java:1138)<br>	at ptolemy.domains.de.kernel.DEDirector._fire(DEDirector.java:1949)<br>	at ptolemy.domains.de.kernel.DEDirector.fire(DEDirector.java:502)<br>	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)<br>	at ptolemy.actor.Manager.iterate(Manager.java:851)<br>	at ptolemy.actor.Manager.execute(Manager.java:369)<br>	at ptolemy.actor.Manager.run(Manager.java:1263)<br>	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)<br>Caused by: javax.script.ScriptException: Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1719)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1764) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1731 at column number 24<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:392)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeMethod(NashornScriptEngine.java:199)<br>	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2281)<br>Caused by: /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1731:24 Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "-32768" is not within "1e-9" of the expected value "32767"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1719)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1764)<br>	at jdk.nashorn.internal.objects.NativeError.initException(NativeError.java:137)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:102)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:106)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:110)<br>	at jdk.nashorn.internal.objects.NativeError.constructor(NativeError.java:129)<br>	at jdk.nashorn.internal.scripts.Script$Recompilation$15589$76254A$require.L:1#react#invokeSpecificHandler(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1736)<br>	at jdk.nashorn.internal.scripts.Script$Recompilation$15588$75900A$require.L:1#react(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1764)<br>	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:639)<br>	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)<br>	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)<br>	at jdk.nashorn.api.scripting.ScriptObjectMirror.callMember(ScriptObjectMirror.java:199)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:386)<br>
</code></td><td>15.646</td>
</tr>
</table>
</body>
</html>
